### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by Ladsgroup**
It seems https://download.sculpin.io/sculpin.phar is broken and
https://sculpin.io/sculpin.phar works instead

### CODE REVIEW

**Comment by thiemowmde** [comment]
There is still a version conflict, but I don't understand why.

**Comment by Ladsgroup** [comment]
I'm working on it.

File: composer.json [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
     "require": {
+	"sculpin/sculpin": "~2.1",
         "worldskills/bootstrap": "~3.2"
     },
     "config": {
         "component-dir": "source/components"
-    }
+    },
+    "minimum-stability": "dev"
```
Comment (JeroenDeDauw) [inline composer.json:9]: Why the min stab?

Comment (Ladsgroup) [reply composer.json:9]: Otherwise it can't install sculpin dependencies. See https://travis-ci.org/wmde/Wikiba.se/builds/241926950

Comment (JeroenDeDauw) [reply composer.json:9]: wtf sculpin...

Comment (JeroenDeDauw) [reply composer.json:9]: Hopefully this works instead: https://github.com/wmde/Wikiba.se/pull/65

Comment (JeroenDeDauw) [reply composer.json:9]: Oh well

**Pull request merged**